test/fixtures/cookbooks/wrapper/recipes/logstash_override.rb

```diff
@@ -1,7 +1,4 @@
 # Attempt to supply an additional logstash configuration file like a real
 # wrapper cookbook would -- see README.md.
-node.set_unless['elkstack']['config']['custom_logstash']['name'] = []
-node.set['elkstack']['config']['custom_logstash']['name'].push('wrapper')
-node.set['elkstack']['config']['custom_logstash']['wrapper']['name'] = 'input_test'
```

Comment:
The main difference, to clarify, is that `name` becomes the same, ie. the name from which the source file is derived if no `source` is given. In the previous approach, the name in the hash key was arbitrary and separate from the name that created the source. Now, they're the same.